pkgs/top-level/perl-packages.nix

```diff
@@ -11946,6 +11946,7 @@ let self = _self // overrides; _self = with self; {
       description = "Networking constants and support functions";
       license = with stdenv.lib.licenses; [ artistic1 gpl1Plus ];
     };
+    doCheck = false;
```

Comment:
Please add a comment explaining why the tests have been disabled.

Reply:
I am not sure myself, it seems that this is a pretty old issue (see 70f0d79d063c665a9fd46d5bd38391f940f0430f), but I am unable to track what happened to those ancient disabled tests.  Is there still a mechanism to disable only selected tests?

Reply:
Nixpkgs doesn't have any "builtin" to do that. You have to fix it yourself.

Reply:
for python we can disable using `py.test`, maybe perl has a similar tool? Otherwise sometimes deleting or patching can solve it.